xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/hibernate.cfg.xml.vm

```diff
@@ -160,7 +160,7 @@
     <property name="hibernate.dbcp.poolPreparedStatements">true</property>
     <property name="hibernate.dbcp.maxOpenPreparedStatements">20</property>
 
-    <property name="hibernate.connection.charSet">UTF-8</property>
+    <property name="hibernate.connection.charSet">utf8mb4</property>
```

Comment:
Not that this change has no impact. This is just a comment. Also from what I tested even if it was in the right place it's not really needed as in both case the connector send the same thing (a String in UTF8), the difference is only on server side.

Reply:
So if I rollback this change, it would ok for you to merge?

Reply:
Sure, looks good to me.

Reply:
We need to decide if we want that in 11.10.x branch too but probably since it's close to a bug.

Reply:
yes please ! (a MySQL bug actually...)